litr/src/main/java/com/linkedin/android/litr/utils/FileUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ */
+package com.linkedin.android.litr.utils;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.ParcelFileDescriptor;
+import androidx.annotation.Nullable;
+import java.io.FileNotFoundException;
+
+public class FileUtils {
+
+    private FileUtils() {}
+
+    /**
+     * Open a raw file descriptor to access data under a URI.
+     */
+    public static ParcelFileDescriptor getParcelFileDescriptor(Context context, Uri outputFileUri) throws FileNotFoundException {
```

Comment:
Technically, after suggested changes this entire class can go away and become a single method call in `MediaMuxerMediaTarget`